### PR TITLE
Added css animation on 404 page

### PR DESCRIPTION
**Pull request by YoganshSharma**

### CODE REVIEW

**Comment by chirag-ghosh** [comment]
image
seems to be some issue.

**Comment by rajivharlalka** [comment]
@chirag-ghosh 
image

**Comment by rajivharlalka** [comment]
[image: image.png]
@chiragghosh
Rajiv Harlalka,
Department of Mathematics
20MA20073,
Phone: +918987966604
Indian Institute of Technology, Kharagpur

File: src/pages/404.astro [hunk 1]
```diff
@@ -31,34 +32,74 @@ import Layout from "../layouts/Layout.astro";
 </Layout>
 
 <script>
-  import { typeIt } from '../utils/typeIt';
+  import { typeIt } from "../utils/typeIt";
+
+  const hackerTextEffect = (target, duration) => {
+    const chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz`~!@#$%^&*(){}:\"|<>?,./;'[]\\=-0987654321";
+    let reqId, start;
+    const step = (timestamp) => {
```
Comment (harshkhandeparkar) [inline src/pages/404.astro:40]: Please add some empty lines for readability :)

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]

**Review by chirag-ghosh — CHANGES_REQUESTED** [review]

File: src/pages/404.astro [hunk 2]
```diff
@@ -31,34 +32,74 @@ import Layout from "../layouts/Layout.astro";
...
+
+      if (progress > duration) {
+        window.cancelAnimationFrame(reqId);
+      } else {
+        reqId = window.requestAnimationFrame(step);
+      }
+    };
+
+    reqId = window.requestAnimationFrame(step);
+  };
```
[the remark this answers is not in the document]
Comment (chirag-ghosh) [reply src/pages/404.astro:56]: Duplicate: https://github.com/kossiitkgp/YAKW/pull/62#discussion_r1316091823

**Review by harshkhandeparkar — APPROVED** [review]
LGTM

**Comment by harshkhandeparkar** [comment]
Merging this because sukha sukha lag raha tha.

**Pull request merged**